script/migration/migrations/20190930200527_version_difficulty.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class VersionDifficulty extends AbstractMigration
+{
+    public function up()
+    {
+        $this->query("
+            ALTER TABLE version
+            ADD difficulty 
+            ENUM('easiest', 'easy', 'more-difficult', 'very-difficult', 'extremely-difficult')
```

Comment:
Why don't we use blue, green, black, 2black here?

Reply:
I don't mind changing it, but I think it's better understood as it is. Many users doesn't know what 'black diamond' means, but when they see 'Very Difficult'... "yes, that track was very difficult".

If the level is set only by the admin (maybe will be better), then blue, green, etc is really better.

Reply:
I think we should use Black diamin, green ... everywhere.
Users are already familiar with these difficulty system from other sites and bike parks.
Later we can include link to difficulty explanations.
For example https://bikepark.borovets-bg.com/?portfolio=machete
> DIFFICULTY: Blue